app/models/vm_migrate_workflow.rb

```diff
@@ -22,6 +22,8 @@ def get_source_and_targets(refresh = false)
     return @target_resource = {} if ems.length != 1
 
     result = {:ems => ci_to_hash_struct(ems.first)}
+    @manager = ems.first
```

Comment:
I don't see any good way of doing it since the `ems` is a result of 
`ems = @values[:src_ids].to_miq_a.collect { |v_id| v = Vm.find_by_id(v_id); v.ext_management_system }.uniq.compact `

which I don't really want to repeat in `field_unsupported`.

But this method anyway have side effects so not sure this is an issue.